docs/release-engineering.md

```diff
@@ -7,6 +7,31 @@ title: How to build and ship a release
 
 Here are my notes from producing the v0.2.8 release.  Change the version number as appropriate.
 
+## Tip: How to keep modules updated
+
+List out-of-date modules and update any that 
+
+```
+go get -u github.com/psampaz/go-mod-outdated
+go list -u -m -json all | go-mod-outdated -update -direct 
+```
+
+To update a module, `get` it, then re-run the integration tests.
+
+```
+go get -u
+or
+go get -u module/path
+```
+
```

Comment:
It would also be a good idea to run `go mod tidy` here to ensure unused dependencies are removed and `go.sum` is up to date.